test/integration/default/inspec/default_spec.rb

```diff
@@ -7,3 +7,31 @@
     it { should be_installed }
   end
 end
+
+describe file('/usr/local/bin/phpcheck') do
+  it { should exist }
+  it { should be_executable }
+end
+
+describe file('/usr/local/bin/phpshow') do
+  it { should exist }
+  it { should be_executable }
+end
+
+describe command('/usr/local/bin/phpcheck -h') do
+  its('stdout') { should match 'phpcheck / phpshow help:' }
+end
+
+describe command('/usr/local/bin/phpshow -h') do
+  its('stdout') { should match 'phpcheck / phpshow help:' }
+end
+
+describe command('/usr/local/bin/phpcheck -d /var/www test for me') do
+  its('stdout') { should match /^phphelpertest$/ }
+  its('stdout') { should match /^test_the_second$/ }
+end
+
+describe command('/usr/local/bin/phpshow -d /var/www test for me') do
```

Comment:
I'd try this one without this to ensure this also works:
```suggestion
describe command('/usr/local/bin/phpshow test for me') do
```